openstack/nova/files/compute.nova.conf

```diff
@@ -17,7 +17,8 @@ lock_path = {{ salt['pillar.get']('nova:lock_path', '/var/lock/nova') }}
 force_dhcp_release = {{ salt['pillar.get']('nova:force_dhcp_release', 'True') }}
 iscsi_helper = {{ salt['pillar.get']('nova:iscsi_helper', 'tgtadm') }}
 
-sql_connection = mysql://nova:{{ salt['pillar.get']('virl:mysql_password', salt['grains.get']('mysql_password', 'password'))}}@{{salt['pillar.get']('virl:internalnet_controller_ip',salt['grains.get']('internalnet_controller_ip', '172.16.10.250'))}}/nova
+# should not be needed on computes
+sql_connection = mysql://nova:{{ virl.mypassword }}@{{ virl.controller_ip }}/nova
```

Comment:
computes should only use api/rpc to get at information; all db-accessing services should be controller-only.